CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic
+Versioning](https://semver.org/spec/v2.0.0.html) for each flavored OpenEdx
+release.
+
+## [Unreleased]
+
+### Added
+
+- First draft of the OpenEdx E-Commerce service docker image based on sources
+  from the `fun/ecommerce-ol` branch of [our `ecommerce`
+  fork](https://github.com/openfun/ecommerce)
```

Comment:
would be nice if this docker image was built on edx's code and the fun branch was targeted via an environment variable in our CI?

Reply:
Yes, that was my main concern while designing this project. I'll try to do this. Even tho, I'm not 100% sure that the same build procedure applies to recent releases.

Reply:
Looks like our Docker image build procedure also applies to recent releases.